app/main.py

```diff
@@ -1,6 +1,28 @@
+import requests
+import os
+
+
+API_KEY = os.getenv("API_KEY")
+if not API_KEY:
+    raise ValueError("API_KEY environment variable is not set.")
+
+URL = f"http://api.weatherapi.com/v1/current.json?key={API_KEY}&q="
+
+FILTER = "Paris"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+
+    title_request = "Performing request to Weather API for city Paris..."
+    result = requests.get(URL + FILTER)
+    data_paris = result.json()
```

Comment:
It would be good to check response code to unsure everything is ok, before doing `json()`